Atomic/push.py

```diff
@@ -71,12 +75,17 @@ def cli(subparser):
     #                      help=_("Organization Name"))
 
 class Push(Atomic):
-    def __init__(self):
-        super(Push, self).__init__()
-
     def push(self):
         self.ping()
-        prevstatus = ""
+        if self.args.debug:
+            util.write_out(str(self.args))
+
+        # This allows a user to turn off signing when a global
+        # sign_by has been defined in /etc/atomic.conf and saves
+        # us from having to define something like --no-sign
+        if self.args.sign_by == "None":
```

Comment:
you can pass None to turn off signing if a default signer is defined in /etc/atomic.conf like the comment above says